receiver/gitmetricsreceiver/README.md

```diff
@@ -0,0 +1,34 @@
+# Git Metrics Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section --> 
+
+The Git Metrics receiver generates metrics from Git about Git repositories and 
+organizations from various vendor endpoints. It is intended to be used
+when the collector is deployed as an agent, scraping over a given interval of
+time.
+
+## Getting Started
+
+The collection interval is defaulted to 30 seconds across the board. 
+
+```yaml
+gitmetrics:
+    collection_interval: <duration> #default = 30s
+    scrapers:
+        <scraper1>:
+        <scraper2>:
+        ...
+```
+
+The available scrapers are:
+| Scraper  | Description             |
+|----------|-------------------------|
+| [github] | Git Metrics from GitHub |
```

Comment:
We assume everyone knows what GitHub is, but let's still put the link there :wink:

```suggestion
| [github] | Git Metrics from [GitHub](https://github.com/) |
```